### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by dipashreepatil**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by RandellDawson** [comment]
Closed and reopened this PR in an attempt to resolve the following Travis build failure:

> AssertionError: The <code>CurrentDate</code> component should render the value from the <code>date</code> prop in the <code>p</code> tag

**Comment by RandellDawson** [comment]
Closed and reopened in an attempt to resolve specific Travis build failure.

File: guide/english/agile/actual-time-estimation/index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Most of the research has focused on the construction of formal software effort e
 
 The perhaps most common estimation methods today are the parametric estimation models COCOMO, SEER-SEM and SLIM. They have their basis in estimation research conducted in the 1970s and 1980s and are since then updated with new calibration data, with the last major release being COCOMO II in the year 2000. The estimation approaches based on functionality-based size measures, e.g., function points, is also based on research conducted in the 1970s and 1980s, but are re-calibrated with modified size measures and different counting approaches, such as the use case points or object points in the 1990s and COSMIC in the 2000s.
 
-Actual time estimation is a time-based method of estimating development work.  
+Actual time estimation is a time-based method of estimating development work. It is used to find out the best approximate time period required to perform the complete work in measures like e.g.certain task will be finished by 31 october or it will take 24 hours for completion .It can be predicted by using time complexities as one of the criterion which can be helped to optimize the code further to reduce the time required.
```
Comment (co16353sidak) [inline guide/english/agile/actual-time-estimation/index.md]: Small Change here:
`31 october` -> `31st October` : better grammar and readability.
`like e.g.certain task` -> `like for e.g certain task`

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@dipashreepatil I have requested some small grammatical changes into this PR, please go through them.

/cc: @RandellDawson

**Review by moT01 — APPROVED** [review]
Looks good enough for now @dipashreepatil - congratulations on your first contribution to freeCodeCamp!

**Pull request merged**